### ISSUE TITLE
Benchmark

### ISSUE THREAD

**Issue by raziel057** [k911/swoole-bundle#66]
It would be great to check the perf improvement by adding the symfony-swoole on https://www.techempower.com like already done or laravel.

**Comment by jderusse** [comment]
This issue can be closed. PR https://github.com/TechEmpower/FrameworkBenchmarks/pull/5237 have been merged

**Comment by k911** [comment]
@jderusse Thanks, I think we can improve it drastically. Setting `running_mode: process` and using https://github.com/swoole/swoole-src#-amazing-runtime-hooks could potentially improve performance. I'll look into it.

**Comment by jderusse** [comment]
Good catch, just switching to `running_mode: reactor` boost performances by ~10% 
Goroutine in `/updates` and `/queries` may be great too. Tell me if you don't have time to do it

**Comment by k911** [comment]
Currently, I'm busy, so feel free to write code

PS. I'll try to look into it over the weekend

**Comment by jderusse** [comment]
I tried to use coroutine, but it don't fit well with doctrine (got "MySQL server has gone away").
Unless I missed a thing, the goal I don't see how to parallelise queries with coroutine.
